arangod/GeneralServer/HttpCommTask.cpp

```diff
@@ -602,14 +600,16 @@ void HttpCommTask<T>::sendResponse(std::unique_ptr<GeneralResponse> baseRes,
   _response = response.stealBody();
   // append write buffer and statistics
   double const totalTime = stat.ELAPSED_SINCE_READ_START();
+  double const queueTime = stat.ELAPSED_WHILE_QUEUED();
 
+  using namespace std::chrono;
   // and give some request information
   LOG_TOPIC("8f555", DEBUG, Logger::REQUESTS)
       << "\"http-request-end\",\"" << (void*)this << "\",\""
       << this->_connectionInfo.clientAddress << "\",\""
-      << GeneralRequest::translateMethod(::llhttpToRequestType(&_parser))
-      << "\",\"" << static_cast<int>(response.responseCode()) << "\","
-      << Logger::FIXED(totalTime, 6);
+      << GeneralRequest::translateMethod(::llhttpToRequestType(&_parser)) << "\",\""
+      << _url << "\",\"" << static_cast<int>(response.responseCode()) << "\",tot:"
+      << Logger::FIXED(totalTime, 6) << ",que:" << Logger::FIXED(queueTime, 6) ;
```

Comment:
the prefixes "tot:" and "que:" make it more difficult to extract the numbers with awk, I would not add them.